contracts/FlowTransactionSchedulerUtils.cdc

```diff
@@ -290,6 +291,7 @@ access(all) contract FlowTransactionSchedulerUtils {
                 ids.remove(at: index!)
                 if ids.length == 0 {
                     self.idsByTimestamp.remove(key: timestamp)
+                    self.sortedTimestamps.remove(timestamp: timestamp)
```

Comment:
Since these were not removed prior, do we need some sort of backfill logic to remove irrelevant sorted timestamps?